srcpkgs/tree-sitter/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'tree-sitter'
+pkgname=tree-sitter
+version=0.17.3
+revision=1
+build_style=gnu-makefile
+short_desc="Incremental parsing system for programming tools"
+maintainer="Gabriel Sanches <gabriel@gsr.dev>"
+license="MIT"
+homepage="https://tree-sitter.github.io/"
+_base_url="https://github.com/${pkgname}/${pkgname}"
```

Comment:
I don't think the variable is necessary, having the full url in changelog without variables is always a lot easier to handle.

Reply:
There is no changelog on the releases page, so listing this is giving a false impression.

Reply:
Sorry, I got a little confused. To which change log are you referring?

Reply:
Nevermind, I just realized there is a `changelog` property. I'll fix this, thanks for the tip!

Reply:
Yes remove the `_base_url` variable and just use the full url and remove the changelog variable as it doesn't provide any useful information.

Reply:
Done,  is it OK the way it is?